src/main/java/org/startsteps/week12/class_01/Flights.java

```diff
@@ -0,0 +1,121 @@
+package org.startsteps.week12.class_01;
+
+import org.startsteps.week12.class_01.persistence.DatabaseUtil;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class Flights {
+
+    public static void insertFlight(String airline, String origin, String destination, String departureTime, String arrivalTime, double price, int seatsAvailable) throws SQLException {
```

Comment:
[Too many parameters](https://wiki.c2.com/?TooManyParameters) 
Potentially pass an object of Flight instead.

Reply:
yes, good catch, this has too many params and could be replaced with an object! 👍